packages/ui/__tests__/Alert.test.tsx

```diff
@@ -97,19 +120,19 @@ describe('Alert', () => {
     expect(closeToast).toHaveBeenCalledTimes(1)
   })
 
-  const alertActionsTestData = [
+  const alertActionsTestData: Array<[string, Array<AlertActionLink | AlertActionButton>]> = [
     ['1 action', [AlertAction1]],
     ['2 actions', [AlertAction1, AlertAction2]],
   ]
 
-  it.each(alertActionsTestData)('Renders Alert with %s', async (_, actions) => {
-    const wrapper = await mountAndCheckA11Y(<Alert type="success" title={title} content={content} closeToast={noOp} />)
+  it.each(alertActionsTestData)('Renders Alert with %s', async (x, actions) => {
```

Comment:
Nit: Just because of the convetion.

```suggestion
  it.each(alertActionsTestData)('Renders Alert with %s', async ( _, actions) => {
```